src/node_api.rs

```diff
@@ -139,6 +143,27 @@ impl Node {
 		}
 	}
 
+	// build an offer to receive payments at this node. path_pubkeys lists the nodes the path will contain,
+	// starting with the introduction node and ending in the destination node (the current node).
+	pub async fn create_offer(
```

Comment:
Out of interest - do you think there's _any_ chance of this being added to `ldk_sample` upstream?

Reply:
Good question! I'll ask